### PR TITLE
🐛(cookiecutter) fix add-site on some situations

### PR DESCRIPTION
**Pull request by igobranco**
Fix some cases where the run of cookiecutter `add-site` Makefile target raises the error:
IOError: [Errno 13] Permission denied: u'/.cookiecutter_replay/template.json'

Full log error.
```bash
site: bar
domain: bar.example.com
Traceback (most recent call last):
  File "/usr/bin/cookiecutter", line 11, in <module>
    sys.exit(main())
  File "/usr/lib/python2.7/site-packages/click/core.py", line 716, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python2.7/site-packages/click/core.py", line 696, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python2.7/site-packages/click/core.py", line 889, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python2.7/site-packages/click/core.py", line 534, in invoke
    return callback(*args, **kwargs)
  File "/usr/lib/python2.7/site-packages/cookiecutter/cli.py", line 106, in main
    config_file=user_config
  File "/usr/lib/python2.7/site-packages/cookiecutter/main.py", line 137, in cookiecutter
    dump(config_dict['replay_dir'], template_name, context)
  File "/usr/lib/python2.7/site-packages/cookiecutter/replay.py", line 37, in dump
    with open(replay_file, 'w') as outfile:
IOError: [Errno 13] Permission denied: u'/.cookiecutter_replay/template.json'
```

### CODE REVIEW

**Comment by igobranco** [comment]
@sampaccoud  I think we should add this line, to minimize the risk during the workshop.

File: cookiecutter/{{cookiecutter.organization}}-richie-site-factory/Makefile [hunk 1]
```diff
@@ -56,6 +56,7 @@ bootstrap:  ## install development dependencies
 .PHONY: bootstrap
 
 add-site: ## add a new site to the site factory
+	@mkdir -p .cookiecutter_replay
 	@docker run --rm -it -e LC_ALL=C.UTF-8 \
```
Comment (sampaccoud) [inline cookiecutter/{{cookiecutter.organization}}-richie-site-factory/Makefile:60]: Oh ok. Then maybe add a `. git` keep file in the `.cookiecutter_replay` directory?

Comment (igobranco) [reply cookiecutter/{{cookiecutter.organization}}-richie-site-factory/Makefile:60]: With that, I think we don't need the line on the Mafile.

Comment (sampaccoud) [reply cookiecutter/{{cookiecutter.organization}}-richie-site-factory/Makefile:60]: @igobranco do you want to change your PR to add the .gitkeep file or do you want me to open another PR?

Comment (igobranco) [reply cookiecutter/{{cookiecutter.organization}}-richie-site-factory/Makefile]: Updated the PR.

**Review by sampaccoud — APPROVED** [review]

**Pull request merged**